apps/dapp/.env.mainnet

```diff
@@ -60,3 +60,6 @@ NEXT_PUBLIC_DISCORD_NOTIFIER_API_BASE=https://discord-notifier.daodao.zone
 
 # Following DAOs
 NEXT_PUBLIC_FOLLOWING_DAOS_API_BASE=https://following.daodao.zone
+
+# Kado API
+NEXT_PUBLIC_KADO_API_KEY=bcee9402-d79f-427b-bafd-dd111e4ce287
```

Comment:
Just checking: This is okay to publish publicly, right?

Reply:
Yep, it's the API key for the iframe, so it's public.